Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -328,6 +328,13 @@ class MessageGenerator {
 
       var ranges = descriptor.extensionRanges.makeIterator()
       var nextRange = ranges.next()
+      var shouldGenerateInclude = false
```

Comment:
This can be sped up since you only need to loop until you find one, something like:

```swift
      if fieldsSortedByNumber.contains { $0.shouldGenerateIncludeDefault } {
        p.print("let shouldIncludeDefault = visitor.shouldIncludeDefault()\n")
      }
```

should do it.